### DISCUSSION TITLE
ADS(R)-Envelope

### DISCUSSION THREAD

**Discussion by banalytic** [publicsamples/Oi-Grandad#38] [Q&A]
Hi!
I am using Oi Grandad V.1.1.5. - In the Guide the Amp-enevelope is described as "ADSR: A simple amplitude envelope" ... I asked me, if there is a Release-Stage?! There is just a slightly difference when changing the decay-parameter. When triggering a low note there is a release while at higher notes there is nothing ... any idea?!. would be so nice to have a dedicated release-parameter... :)
Thanks for this great Plugin anyway!!!

**Comment by RickTaylor79** [comment]
There is no proper release stage unfortunately, this is a quirk of the HISE granular player and not something I can easily overcome.

This is partially why each voice has it's own delay, with a bit of careful programming you can overcome this somewhat.. In Oi, Grandad! v2 this is improved on slightly by placing an ADSR after the delay/resonator stage.

**Reply by banalytic** [reply]
thanks for fast reply! So I will install V2 and have a look… :)
all the best!
